README.md

```diff
@@ -30,3 +30,27 @@ To make all this config work, you only need to add this to `package.json`:
   }
 }
 ```
+
+## Migrate from v2?
+
+We removed `jest`-related rules in v3, essentially making `jest` an optional
+dependency. In our experience, we don't necessarily want to use Jest for every
+project anymore. Versions below v3 **require** `jest`, which just doesn't make
+sense to us.
+
+However, there are a few steps needed to get it working anyway:
```

Comment:
This line reads to me like: "There were already some steps required, so dropping it doesn't really make much of a difference"

What about changing it to something like: "Here are the steps to get it working [again]:", or just "To get it working [again], do the following{.,:}"?